dev/run-python-flavor-tests.sh

```diff
@@ -12,6 +12,12 @@ pytest tests/pyfunc --large
 pytest tests/azureml --large
 pytest tests/models --large
 pytest tests/utils/test_model_utils.py --large
+
+# TODO: Run tests for h2o, shap, and paddle in the cross-version-tests workflow
+pytest tests/h2o --large
+pytest tests/shap --large
+pytest tests/paddle --large
```

Comment:
We need to keep these until the migration to the cross-version-tests workflow is completed.